engine/security/apparmor.md

```diff
@@ -46,7 +50,11 @@ profile docker-default flags=(attach_disconnected,mediate_deleted) {
   file,
   umount,
 
-  deny @{PROC}/{*,**^[0-9*],sys/kernel/shm*} wkx,
+  deny @{PROC}/* w,   # deny write for all files directly in /proc (not in a subdir)
```

Comment:
Agreed. What do you think if I keep a snippet here to show general syntax and then provide a link to the full profile? Something like this https://docker.github.io/engine/security/seccomp/#passing-a-profile-for-a-container